### PR TITLE
add reference to glamorous

### PR DESCRIPTION
**Pull request by kentcdodds**

### CODE REVIEW

File: docs/tutorial/part-two/index.md [hunk 1]
```diff
@@ -602,4 +604,4 @@ Gatsby supports almost every possible styling option (if there isn't a plugin ye
 * [Emotion](/packages/gatsby-plugin-emotion/)
 * [JSS](/packages/gatsby-plugin-jss/)
 * [Stylus](/packages/gatsby-plugin-stylus/)
-* and more!
\ No newline at end of file
+* and more!
```
Comment (kentcdodds) [inline docs/tutorial/part-two/index.md]: Not sure how this happened 🤷‍♂️

File: docs/tutorial/part-two/index.md [hunk 2]
```diff
@@ -496,6 +496,8 @@ The final Glamor page should look identical to the CSS Modules page.
 
 ![glamor-example](glamor-example.png)
 
+In addition, you can also use this plugin to make [`glamorous`](https://github.com/paypal/glamorous) 💄 work with server side rendering and start writing React components that carry their styles with them.
```
Comment (KyleAMathews) [inline docs/tutorial/part-two/index.md]: Oh sorry, my tweet wasn't specific enough — add the mention of Glamorous to https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-glamor/README.md and then add a Glamorous link along with links to Emotion, JSS, Stylus, etc.

**Comment by KyleAMathews** [comment]
Moved your docs over to the Glamor plugin README. Thanks for contributing!

**Comment by kentcdodds** [comment]
Ah! Thank you! This has just been sitting around in my inbox. I've been filling myself I'd find time to update it 🙃

Thank you!

**Comment by KyleAMathews** [comment]
:-) like you say on Twitter, pretty trivial/convenient to push changes to PRs as a maintainer!

**Comment by jlengstorf** [comment]
Hiya @kentcdodds! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**